installer/targets/tue-save-map/setup

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+
+function _parse_yaml_map_server {
```

Comment:
Why are functions and utilities like these locked up in tue-env? This is not an environment function but something for manipulating maps.